cli-commands/forgae-deploy/forgae-deployer.js

```diff
@@ -99,10 +109,70 @@ class Deployer {
         }
 
         logStoreService.logAction(info);
-
         console.log(`===== Contract: ${contractFileName} has been deployed =====`)
         return deployedContract;
+
+        function addFromFunction (obj) {
```

Comment:
please rename obj to something more meaningful

Reply:
if this function accept ANY object and assign to it this functionality, what is your suggestion for better name?

Reply:
This functions will probably be used only for the deployer, so probably something like deployerInstance or contractInstance will be more meaningful? What do you think?